src/app/shared/entry/info-tab-topic/edit-topic/edit-topic-dialog.component.html

```diff
@@ -57,22 +58,48 @@ <h1 mat-dialog-title>Select a Topic</h1>
           </div>
         </mat-radio-button>
       </mat-card>
-      <mat-card>
-        <mat-radio-button [value]="TopicSelectionEnum.AI" data-cy="topic-ai-selection-button">
-          <div>
-            <span class="bubble mr-2"><img class="site-icons-small" src="../../../assets/svg/ai-icon.svg" alt="AI generation icon" /></span>
-            <strong>AI</strong>
+      <mat-card *ngIf="entry.topicAI">
```

Comment:
New: The AI option is only displayed if there is an AI topic. This is to prevent the user from approving an AI topic that doesn't exist